src/middlewares/userMiddlewares.js

```diff
@@ -27,7 +28,7 @@ export default class UserMiddlewares {
   }
 
   /**
- * @name checkUserExistBeforeLogin
+ * @name doesUserExists
```

Comment:
Why did you change the name of the method in jsdoc.

Reply:
I changed it initially but decided to leave it the way it was since it was merged to develop already

Reply:
Please change the jsdoc name back to what it should be